### DISCUSSION TITLE
Public Dashboard?

### DISCUSSION THREAD

**Discussion by lemmy-music** [kizniche/Mycodo#1363] [Q&A]
Hi, is there a possibility to set a password to "public" so that you can access and view it without any login, but you can not edit anything in the software? (only a simple view)

**Comment by kizniche** [comment]
Yes, on the Users page you can set permissions.

**Reply by lemmy-music** [reply]
Thank you very much, the permisisons do work, but can i somehow make it possible to view a dashboard without a login?
I want to provide the dashboard only to friends, so they can easily monitor my plants, if they are interested, by just clicking a link.

**Reply by lemmy-music** [reply]
I tried passing the credentials in the URL, but that did not work aswell

**Reply by kizniche** [reply]
No, credentials are required for the permission system to work.
